app/controllers/answers_controller.rb

```diff
@@ -13,10 +13,36 @@ def new
 
   def create
     @question = Question.find params[:id]
+
     @answer = Answer.new(answer_params.merge(parent: @question, user: current_user, score: 0,
-                                             body: helpers.render_markdown(params[:answer][:body_markdown]),
-                                             last_activity: DateTime.now, last_activity_by: current_user,
-                                             category: @question.category))
+      body: helpers.render_markdown(params[:answer][:body_markdown]),
+      last_activity: DateTime.now, last_activity_by: current_user,
+      category: @question.category))
```

Comment:
These should line up with the line above:

```suggestion
                                             body: helpers.render_markdown(params[:answer][:body_markdown]),
                                             last_activity: DateTime.now, last_activity_by: current_user,
                                             category: @question.category))
```